src/renderer/components/RemoveServerModal.tsx

```diff
@@ -26,7 +25,7 @@ function RemoveServerModal(props: Props) {
             acceptLabel={intl.formatMessage({id: 'label.remove', defaultMessage: 'Remove'})}
             cancelLabel={intl.formatMessage({id: 'label.cancel', defaultMessage: 'Cancel'})}
             body={(
-                <Modal.Body>
+                <>
```

Comment:
Would this affect any styling for this modal?

Reply:
Nope, I checked and it all looks fine

Reply:
Nope, I checked and it all looks fine